harvest/trader/trader.py

```diff
@@ -83,27 +82,12 @@ def __init__(self, streamer=None, broker=None, storage=None, debug=False):
         self.server = Server(self)
 
         # Set up logger
-        self.debugger = logging.getLogger("harvest")
-        self.debugger.setLevel("DEBUG")
         if debug:
-            f_handler = logging.FileHandler("trader.log")
-            f_handler.setLevel(logging.DEBUG)
-            f_format = logging.Formatter(
-                "%(asctime)s : %(name)s : %(levelname)s : %(message)s"
-            )
-            f_handler.setFormatter(f_format)
-            self.debugger.addHandler(f_handler)
-
-        c_handler = logging.StreamHandler()
-        if debug:
-            c_handler.setLevel(logging.DEBUG)
-        else:
-            c_handler.setLevel(logging.INFO)
-        c_format = logging.Formatter(
-            "%(asctime)s : %(name)s : %(levelname)s : %(message)s"
+            debugger.setLevel("DEBUG")
+
+        debugger.debug(
+            f"Streamer: {type(self.streamer).__name__}\nBroker: {type(self.broker).__name__}\nStorage: {type(self.storage).__name__}"
         )
-        c_handler.setFormatter(c_format)
-        self.debugger.addHandler(c_handler)
```

Comment:
logger defined in utils so that it can be used without needed to have the trader. Useful if someone just wants to use a broker.